core/domain/feedback_services.py

```diff
@@ -179,7 +182,7 @@ def get_last_updated_time(exploration_id):
 
     If this exploration has no threads, returns None.
     """
-    threadlist = get_all_threads(exploration_id, False)
+    threadlist = get_displayable_threads(exploration_id, False)
     return max(
```

Comment:
Ah, I see -- you're saying that the original code was wrong. I think that is the case, actually.

Having said that, I traced this back to the frontend, and I found that this function is only being used by core.controllers.feedback, in a handler that never actually gets called by the frontend. Can you confirm this? If so, maybe we should just get rid of it (and the corresponding unused stuff in the controller, feconf.py and main.py)?